types/types.go

```diff
@@ -17,3 +17,7 @@ type CensusRoot = []byte
 type EncodedProtoBuf = []byte
 
 type Nullifier = []byte
+
+type FromAccount = []byte
```

Comment:
Why this type redefinition? Is this a requirement for sqlite?

Reply:
Not a strong requirement but following @mvdan approach

Reply:
I'd avoid this if its not required.

Reply:
this is currently required because sqlc will otherwise generate `string` as a type for the sql `BLOB`, which will cause problems - strings in SQL are null-terminated, for example, so then any query containing a null byte in a blob will be truncated.

Reply:
at the same time, assuming FromAccount and ToAccount are both the same type, perhaps use `type AccountID = []byte`, then we save on the repetition.